### PR TITLE
Added error messages to the cashier page

### PR DESCRIPTION
**Pull request by ethanphunter**
Resolves: https://github.com/ethanphunter/PointOfSaleProject/issues/70

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -12,7 +13,7 @@
 products = []
 
 app.config.update(dict(
-    #DEBUG=True,
+    DEBUG=True,
```
Comment (calvinbrown085) [inline main.py]: @ethanphunter You left debug to true

**Comment by calvinbrown085** [comment]
@ethanphunter Good other than my comment

File: main.py (hunk 1 above)
Comment (ethanphunter) [inline main.py]: Dang it, I made sure I got the key fixed but not debug...

**Comment by ethanphunter** [comment]
@calvinbrown085 Done

**Comment by calvinbrown085** [comment]
👍

**Comment by gilmerc** [comment]
:+1:

**Pull request merged**